test/index-test.js

```diff
@@ -202,3 +203,59 @@ test('vue', (t) => {
     t.end();
   }).catch(t.end);
 });
+
+const unparsedHtmlExpectedWarnings = [
+  {
+    line: 15,
+    column: 12,
+    rule: 'CssSyntaxError',
+    severity: 'error',
+    text: 'Unknown word (CssSyntaxError)',
+  },
+];
+
+test('only files that match the regex filter should pass through the processor', (t) => {
+  const fixtureOne = path.join(__dirname, './fixtures/markdown.md');
+  const fixtureTwo = path.join(__dirname, './fixtures/html.html');
+
+  stylelint.lint({
+    files: [fixtureOne, fixtureTwo],
+    config: {
+      processors: [[pathToProcessor, { fileFilterRegex: [/\.md$/] }]],
+      rules: config.rules,
+    },
+  }).then((data) => {
+    t.equal(data.results.length, 2, 'number of results');
+
+    t.equal(data.results[0].source, fixtureOne);
+    t.deepEqual(_.orderBy(data.results[0].warnings, ['line', 'column']), markdownExpectedWarnings);
+
+    t.equal(data.results[1].source, fixtureTwo);
+    t.deepEqual(_.orderBy(data.results[1].warnings, ['line', 'column']), unparsedHtmlExpectedWarnings);
+
+    t.end();
+  }).catch(t.threw);
+});
```

Comment:
Can you please add a test where you have multiple items in the `fileFilterRegex` array?

Reply:
Sure! 👍